Mergin/plugin.py

```diff
@@ -717,8 +717,9 @@ def sync_project(self):
 
     def _reload_project(self):
         """This will forcefully reload the QGIS project because the project (or its data) may have changed"""
+        qgis_proj_filename = os.path.normpath(QgsProject.instance().fileName())
         qgis_files = find_qgis_files(self.path)
-        if QgsProject.instance().fileName() in qgis_files:
+        if qgis_proj_filename in qgis_files:
             iface.addProject(QgsProject.instance().fileName())
```

Comment:
This whole `_reload_project()` method seems unused. Could we remove it or am I missing something?

Reply:
Indeed it is unused I removed it in the last commit

Reply:
Thanks for the catch